wordpress-mobile/WordPress-iOS
# Go back to bumping site created

Fixes #11809 

## Testing:

For all tests:

- Place a breakpoint [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/c2d19fda695d6b8d94bba230aaf9e3db4e53826f/WordPress/Classes/ViewRelated/Site%20Creation/FinalAssembly/SiteAssemblyWizardContent.swift#L118)
- Place a breakpoint [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/c2d19fda695d6b8d94bba230aaf9e3db4e53826f/WordPress/Classes/ViewRelated/Site%20Creation/FinalAssembly/SiteAssemblyWizardContent.swift#L216)

### Test 1:

1. Create a Site.
2. Make sure the first breakpoint is triggered right after the site is created, before the preview of it is shown.
3. Make sure the second breakpoint is triggered when you tap "Done" in the site preview screen.

### Test 2:

Same as the test above but make sure that if you disconnect from internet right before you confirm the site creation, neither breakpoint is triggered.

Update release notes:
